servers/src/test/java/tachyon/worker/block/meta/StorageDirTest.java

```diff
@@ -43,7 +43,7 @@
   private TempBlockMeta mTempBlockMeta;
 
   @Rule
-  public ExpectedException thrown = ExpectedException.none();
+  private ExpectedException mThrown = ExpectedException.none();
```

Comment:
ok. All members either public or private should start with 'm', a bit confusing, I thought only private members should start with 'm'.